src/lib/__tests__/getCoverage.test.ts

```diff
@@ -8,4 +9,42 @@ describe("getCoverage function", () => {
 
     done();
   });
+
+  it("tsProjectFile is passed into lint function, if provided", async (done) => {
+    typeCoverageCore.lint = jest.fn().mockImplementation(typeCoverageCore.lint);
+
+    const data = await getCoverage({
+      tsProjectFile: "./app/tsconfig.json"
+    });
```

Comment:
Don't need to test for this (line `20`). Just run

```js
await getCoverage({
```

Reply:
I'm not totally sure what you mean here - sorry. Have updated to best guess!